### PR TITLE
issue #7201

### PR DESCRIPTION
**Pull request by rachana-webkul**
## Issue Reference
#7201

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
This is **working fine.** Please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63e0d167aac9534fac239361)

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Review by devansh-webkul — APPROVED** [review]

File: packages/Webkul/Product/src/Type/Configurable.php [hunk 1]
```diff
@@ -222,12 +224,13 @@ public function createVariant($product, $permutation, $data = [])
...
-                'price'       => 0,
-                'weight'      => 0,
-                'status'      => 1,
+                'sku'               => $product->sku . '-variant-' . implode('-', $permutation),
+                'name'              => '',
+                'inventories'       => [],
+                'price'             => 0,
+                'weight'            => 0,
+                'status'            => 1,
+                'tax_category_id'   => ''
```
Comment (jitendra-webkul) [inline packages/Webkul/Product/src/Type/Configurable.php:233]: Remove extra spaces

Comment (jitendra-webkul) [inline packages/Webkul/Product/src/Type/Configurable.php:233]: Add comma after last array element

**Review by devansh-webkul — DISMISSED** [review]

File: packages/Webkul/Product/src/Type/Configurable.php [hunk 2]
```diff
@@ -198,6 +199,7 @@ public function update(array $data, $id, $attribute = 'id')
 
                     $variantData['locale'] = $data['locale'];
 
+                    $variantData['tax_category_id']=$data['tax_category_id'];
```
Comment (jitendra-webkul) [inline packages/Webkul/Product/src/Type/Configurable.php:202]: Alway use space before and after with binary operator. And also add new line after this statement

**Comment by rachana-webkul** [comment]
@jitendra-webkul 
spaces added, please verify

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**